test/unittests/parser_test.cpp

```diff
@@ -11,10 +11,30 @@ static const auto functype_i32_to_void = from_hex("60017f00");
 
 namespace
 {
-bytes make_section(uint8_t id, const bytes& content)
+bytes prefix_with_size(const bytes& content)
 {
     assert(content.size() < 0x80);
-    return bytes{id, static_cast<uint8_t>(content.size())} + content;
+    return bytes{static_cast<uint8_t>(content.size())} + content;
+}
+
+bytes make_vec(std::vector<bytes_view> contents)
+{
+    assert(contents.size() < 0x80);
+    bytes ret{static_cast<uint8_t>(contents.size())};
+    for (const auto& content : contents)
+        ret.append(content);
+    return ret;
+}
+
+bytes make_section(uint8_t id, const bytes& content)
+{
+    return bytes{id} + prefix_with_size(content);
+}
+
+bytes make_invalid_section(uint8_t id, size_t size, const bytes& content)
```

Comment:
maybe
```suggestion
bytes make_invalid_size_section(uint8_t id, size_t size, const bytes& content)
```

Reply:
Same here, no strong preference myself. @chfast ?

Reply:
I line suggested name better. But considering number of uses, I'm not convinced it is worth for add this helper in general.

Reply:
I think we'll need to add more tests with invalid sizes to ensure the parser is good, so it will be needed.

Reply:
Changed.